src/components/DropdownMenu/DropdownMenu.tsx

```diff
@@ -0,0 +1,18 @@
+import { Person } from '../../types/Person';
+import { DropdownItem } from '../DropdownItem';
+
+interface Props {
+  people: Person[];
+}
+
+export const DropDownMenu: React.FC<Props> = ({ people }) => {
+  return (
+    <div className="dropdown-menu" role="menu" data-cy="suggestions-list">
+      <div className="dropdown-content">
+        {people.map((person, index) => (
+          <DropdownItem person={person} key={index} />
```

Comment:
You're generating keys based on the index in the map function. This is considered an anti-pattern in React because it can lead to unexpected behavior when items are added, removed, or reordered in the list. Instead, try to use unique and stable identifiers from your data as keys. You can read more about this issue [here](https://medium.com/blackrock-engineering/5-common-mistakes-with-keys-in-react-b86e82020052).